packages/gatsby-theme-docs/src/components/top-menu.js

```diff
@@ -169,7 +177,7 @@ const TopMenu = () => {
           event.stopPropagation();
         }}
       >
-        <ContentGrid>
+        <div css={props.centered ? centeredContainerStyle : contentGridStyle}>
```

Comment:
@emmenko the grid still remains but through css prop now.

Reply:
But the centered container style is not a grid 🧐

Reply:
I'm not sure what you mean, but currently, the children's layout is not affected from what I see - I get exactly what I want by making it a block element, and children are properly centered.